src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,163 @@
+                .collect(Collectors.toList());
+        return initCardForPlayers;
+    }
+
+    private void printInitHands(List<Name> names, Dealer dealer, Players players) {
+        OutputView.printInitMessage(names.stream().map(Name::getName).collect(Collectors.toList()));
+        OutputView.printParticipantStatus(dealer.showOneHand(), players.showHands());
+    }
+
+    private void printDealerBlackJackResult(List<Name> names, Dealer dealer, Players players) {
+        Result dealerBlackjackResult = new Result(players.getResultAtDealerBlackJack(dealer));
+        OutputView.printDealerBlackJackMessage();
+        OutputView.printResultTitle();
+        OutputView.printDealerResult(
+                dealerBlackjackResult.getDealerWinCount(),
+                dealerBlackjackResult.getDealerDrawCount(),
+                dealerBlackjackResult.getDealerLoseCount()
+        );
+        for (Name name : names) {
+            OutputView.printPlayerResult(name.getName(), dealerBlackjackResult.getVersusOfPlayer(name).getResult());
+        }
+    }
+
+    private void printBlackJackPlayer(List<Name> names, Players players) {
+        for (Name name : names) {
+            if (players.isScore21ByName(name)) {
+                OutputView.printPlayerBlackJackMessage(name.getName());
+            }
+        }
+    }
+
+    private void drawForPlayers(List<Name> names, Deck deck, Players players) {
+        for (Name name : names) {
+            if (players.isBlackJackByName(name)) {
+                continue;
+            }
+            askAndDrawForPlayer(deck, players, name);
+        }
+    }
+
+    private void askAndDrawForPlayer(Deck deck, Players players, Name name) {
+        boolean isKeepDraw = true;
+        while (isKeepDraw && inputAskDraw(name.getName())) {
+            players.addCardByName(name, deck.draw());
+            OutputView.printHand(players.showHandByName(name));
+            isKeepDraw = checkScore21OrBust(players, name);
+        }
+    }
+
+    private boolean inputAskDraw(String name) {
+        String askDraw = InputView.inputAskDraw(name);
+        validateAskDraw(askDraw);
+        if (askDraw.equals("y")) {
+            return true;
+        }
+        return false;
+    }
+
+    private void validateAskDraw(String resultAsk) {
+        if (!(resultAsk.equals("y") || resultAsk.equals("n"))) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    private boolean checkScore21OrBust(Players players, Name name) {
+        if (players.isScore21ByName(name)) {
+            OutputView.printBlackJackMessage();
+            return false;
+        }
+        if (players.isBustByName(name)) {
+            OutputView.printBustMessage();
+            return false;
+        }
+        return true;
+    }
+
+    private void drawForDealer(Deck deck, Dealer dealer, Players players) {
+        if (!players.isAllBust()) {
+            checkAndDrawForDealer(deck, dealer);
+        }
```

Comment:
```suggestion
        if (players.isNotAllBust()) {
            checkAndDrawForDealer(deck, dealer);
        }
```
부정 조건을 위해 메서드가 만들어졌다면, 이런형태도 읽기 좋은 것 같아요 🙂

Reply:
딜러가 카드를 더 뽑아야하는지 확인 하는 메서드 `isNeedToDraw()` 도 같은 방향으로 수정했습니다🔧